.github/workflows/push_pr.yml

```diff
@@ -25,4 +25,4 @@ jobs:
   compile_and_test:
     uses: apache/flink-connector-shared-utils/.github/workflows/ci.yml@ci_utils
     with:
-      flink_version: 1.16.0
+      flink_version: 1.16.1
```

Comment:
We should split this out to a separate PR. We can merge this independently

Reply:
@dannycranmer https://github.com/apache/flink-connector-opensearch/pull/7